### DISCUSSION TITLE
TypeScript with shebang misclassified as JavaScript

### DISCUSSION THREAD

**Discussion by isitreallyalive** [github-linguist/linguist#5698] [Classification] [answered]
[This](https://github.com/newtykins/snipe-yourself/blob/48689efac951e94533ce20d340e4fdca2d80de0e/src/index.ts) appears to be the file at hand. It is causing my [snipe-yourself](https://github.com/newtykins/snipe-yourself/tree/48689efac951e94533ce20d340e4fdca2d80de0e) repo to be incorrectly flagged as JavaScript - any idea why?

**Comment by Nixinova** [comment, marked as answer]
See #4558  #5249 #5251

`#!/usr/bin/env node` makes Linguist classify it as Node (JavaScript)
